src/families/ripple/bridge/libcore.js

```diff
@@ -133,6 +134,22 @@ const prepareTransaction = async (a, t) => {
   return t;
 };
 
+const estimateMaxSpendable = async ({
+  account,
+  parentAccount,
+  transaction
+}) => {
+  const mainAccount = getMainAccount(account, parentAccount);
+  const t = await prepareTransaction(mainAccount, {
+    ...createTransaction(),
+    ...transaction,
+    useAllAmount: true,
+    recipient: "rHsMGQEkVNJmpGWs8XUBoTBiAAbwxZN5v3" // public testing seed abandonx11,about
```

Comment:
If all currencies will have this, wouldn't it make sense to have this in `cryptocurrencies.js` in case we want to reuse it somewhere else?

Reply:
I think `cryptocurrencies.js` is meant to be general data of cryptos, here it feels a bit specific to our internal algorithm of dry run estimation. not all bridge implementation & currency will need real addresses (e.g. ETH can just use 0x000...0000).

it's too implementation-specific at the moment to consider generalisation imo.

Reply:
👍